[Bug]: Proxmox script - LXC Creation fails on Line 29

### Preflight Checklist

- [x] I have searched [existing issues](https://github.com/moltis-org/moltis/issues?q=is%3Aissue+label%3Abug) and this hasn't been reported yet
- [x] I am using the latest version of Moltis
- [ ] If this happened during a chat session, I included as much full session context as possible and redacted secrets

### What happened?

When running the helper script and choosing to install on a LXC, it fails with the following error:

  ⚙️  Using Default Settings on node pve1
  💡  PVE Version 9.1.9 (Kernel: 6.17.2-1-pve)
  🆔  Container ID: 102
  🖥️  Operating System: debian (13)
  📦  Container Type: Unprivileged
  💾  Disk Size: 20 GB
  🧠  CPU Cores: 2
  🛠️  RAM Size: 2048 MiB
  🚀  Creating a Moltis LXC using the above default settings
  
  ✔️   Storage local (Free: 81.7GB  Used: 7.5GB) [Template]
  ✔️   Storage local-lvm (Free: 213.7GB  Used: 124.2GB) [Container]
  ✔️   Storage 'local-lvm' (lvmthin) validated
  ✔️   Template storage 'local' validated
  ✔️   Template search completed
  ✔️   Template debian-13-standard_13.1-2_amd64.tar.zst [local]
  ✔️   LXC Container 102 was successfully created.
  ✔️   Started LXC Container
  ✔️   Network in LXC is reachable (ping)
  ✔️   Customized LXC Container
  ✔️   Set up Container OS
  ✔️   Network Connected: 10.10.99.229
  ✔️   IPv4 Internet Connected
  ✖️   IPv6 Internet Not Connected
  ✔️   Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
  ✔️   Updated Container OS
  ✔️   Installed Dependencies
  ✔️   Installed Moltis 20260510.01
  ✖️   in line 29: exit code 1 (General error / Operation not permitted): while executing command read -r -p "${TAB3}Would you like to install Docker for sandbox support? <y/N> " prompt

### Expected behavior

Continuation of the setup process

### Steps to reproduce

1. Run install script on PVE

### Did this happen during a chat session?

None

### Chat session context (if applicable)

```markdown

```

### Error messages / logs

```shell
Get:1 http://deb.debian.org/debian trixie/main amd64 libgomp1 amd64 14.2.0-19 [137 kB]
Fetched 137 kB in 0s (3476 kB/s)
Selecting previously unselected package libgomp1:amd64.
(Reading database ... 20272 files and directories currently installed.)
Preparing to unpack .../libgomp1_14.2.0-19_amd64.deb ...
Unpacking libgomp1:amd64 (14.2.0-19) ...
Setting up libgomp1:amd64 (14.2.0-19) ...
Processing triggers for libc-bin (2.41-12+deb13u2) ...
[2026-05-11 14:01:57] [OK] Installed Dependencies
[2026-05-11 14:01:57] [INFO] Installing Moltis 20260510.01
Selecting previously unselected package moltis.
(Reading database ... 20275 files and directories currently installed.)
Preparing to unpack /tmp/moltis.deb ...
Unpacking moltis (20260510.01) ...
Setting up moltis (20260510.01) ...
[2026-05-11 14:01:59] [OK] Installed Moltis 20260510.01
[2026-05-11 14:01:59] [ERROR] in line 29: exit code 1 (General error / Operation not permitted): while executing command read -r -p "${TAB3}Would you like to install Docker for sandbox support? <y/N> " prompt
```

### Is this a regression?

I don't know

### Last working version

_No response_

### Moltis version

20260510.01

### Component

Configuration

### Install method

Pre-built binary

### Operating system

Ubuntu / Debian

### Additional context

_No response_

--- a/install.sh
+++ b/install.sh
@@ -498,18 +498,18 @@ install_proxmox() {
     # and make the Docker prompt safe when lxc-attach has no interactive stdin.
     awk -v repo_url="$PROXMOX_REPO_URL" '
         {
-            print
             if ($0 ~ /^source <\(curl -fsSL / && !curl_patched) {
                 print "curl() {"
                 print "  case \"${*: -1}\" in"
                 print "    */install/moltis-install.sh)"
-                print "      command curl \"$@\" | sed '\''s/read -r -p \"${TAB3}Would you like to install Docker for sandbox support? <y\\/N> \" prompt/if [[ -t 0 ]]; then read -r -p \"${TAB3}Would you like to install Docker for sandbox support? <y\\/N> \" prompt; else prompt=\"${MOLTIS_INSTALL_DOCKER:-no}\"; fi/'\''"
+                print "      command curl \"$@\" | sed '\''s|^[[:space:]]*read -r -p \".*Docker for sandbox support.*\" prompt$|if [[ -t 0 ]]; then &; else prompt=\"${MOLTIS_INSTALL_DOCKER:-no}\"; fi|'\''"
                 print "      ;;"
                 print "    *) command curl \"$@\" ;;"
                 print "  esac"
                 print "}"
                 curl_patched = 1
             }
+            print
             if ($0 == "description" && !patched) {
                 q = sprintf("%c", 39)
                 print "pct exec \"$CTID\" -- bash -c \"cat > /usr/bin/update <<" q "MOLTIS_UPDATE_EOF" q
@@ -520,6 +520,9 @@ install_proxmox() {
             }
         }
     ' "$proxmox_script" >"$patched_script"
+
+    grep -q 'curl()' "$patched_script" || error "Failed to apply Docker prompt patch to Proxmox helper"
+    grep -q 'MOLTIS_UPDATE_EOF' "$patched_script" || error "Failed to apply update URL patch to Proxmox helper"
 
     info "Launching Proxmox VE helper script..."
     bash "$patched_script"

--- a/install.sh
+++ b/install.sh
@@ -493,12 +493,23 @@ install_proxmox() {
 
     download "$PROXMOX_SCRIPT_URL" "$proxmox_script" || error "Failed to download Proxmox helper"
 
-    # The upstream community-scripts install helper currently writes a hardcoded
-    # /usr/bin/update URL. Patch the fetched helper so new Moltis LXCs update
-    # from the Moltis fork until the helper honors COMMUNITY_SCRIPTS_URL there.
+    # Patch the fetched helper at launch time for Moltis-specific fixes that live
+    # in remote community-scripts files: keep /usr/bin/update on the Moltis fork,
+    # and make the Docker prompt safe when lxc-attach has no interactive stdin.
     awk -v repo_url="$PROXMOX_REPO_URL" '
         {
             print
+            if ($0 ~ /^source <\(curl -fsSL / && !curl_patched) {
+                print "curl() {"
+                print "  case \"${*: -1}\" in"
+                print "    */install/moltis-install.sh)"
+                print "      command curl \"$@\" | sed '\''s/read -r -p \"${TAB3}Would you like to install Docker for sandbox support? <y\\/N> \" prompt/if [[ -t 0 ]]; then read -r -p \"${TAB3}Would you like to install Docker for sandbox support? <y\\/N> \" prompt; else prompt=\"${MOLTIS_INSTALL_DOCKER:-no}\"; fi/'\''"
+                print "      ;;"
+                print "    *) command curl \"$@\" ;;"
+                print "  esac"
+                print "}"
+                curl_patched = 1
+            }
             if ($0 == "description" && !patched) {
                 q = sprintf("%c", 39)
                 print "pct exec \"$CTID\" -- bash -c \"cat > /usr/bin/update <<" q "MOLTIS_UPDATE_EOF" q

--- a/website/install.sh
+++ b/website/install.sh
@@ -498,18 +498,18 @@ install_proxmox() {
     # and make the Docker prompt safe when lxc-attach has no interactive stdin.
     awk -v repo_url="$PROXMOX_REPO_URL" '
         {
-            print
             if ($0 ~ /^source <\(curl -fsSL / && !curl_patched) {
                 print "curl() {"
                 print "  case \"${*: -1}\" in"
                 print "    */install/moltis-install.sh)"
-                print "      command curl \"$@\" | sed '\''s/read -r -p \"${TAB3}Would you like to install Docker for sandbox support? <y\\/N> \" prompt/if [[ -t 0 ]]; then read -r -p \"${TAB3}Would you like to install Docker for sandbox support? <y\\/N> \" prompt; else prompt=\"${MOLTIS_INSTALL_DOCKER:-no}\"; fi/'\''"
+                print "      command curl \"$@\" | sed '\''s|^[[:space:]]*read -r -p \".*Docker for sandbox support.*\" prompt$|if [[ -t 0 ]]; then &; else prompt=\"${MOLTIS_INSTALL_DOCKER:-no}\"; fi|'\''"
                 print "      ;;"
                 print "    *) command curl \"$@\" ;;"
                 print "  esac"
                 print "}"
                 curl_patched = 1
             }
+            print
             if ($0 == "description" && !patched) {
                 q = sprintf("%c", 39)
                 print "pct exec \"$CTID\" -- bash -c \"cat > /usr/bin/update <<" q "MOLTIS_UPDATE_EOF" q
@@ -520,6 +520,9 @@ install_proxmox() {
             }
         }
     ' "$proxmox_script" >"$patched_script"
+
+    grep -q 'curl()' "$patched_script" || error "Failed to apply Docker prompt patch to Proxmox helper"
+    grep -q 'MOLTIS_UPDATE_EOF' "$patched_script" || error "Failed to apply update URL patch to Proxmox helper"
 
     info "Launching Proxmox VE helper script..."
     bash "$patched_script"

--- a/website/install.sh
+++ b/website/install.sh
@@ -493,12 +493,23 @@ install_proxmox() {
 
     download "$PROXMOX_SCRIPT_URL" "$proxmox_script" || error "Failed to download Proxmox helper"
 
-    # The upstream community-scripts install helper currently writes a hardcoded
-    # /usr/bin/update URL. Patch the fetched helper so new Moltis LXCs update
-    # from the Moltis fork until the helper honors COMMUNITY_SCRIPTS_URL there.
+    # Patch the fetched helper at launch time for Moltis-specific fixes that live
+    # in remote community-scripts files: keep /usr/bin/update on the Moltis fork,
+    # and make the Docker prompt safe when lxc-attach has no interactive stdin.
     awk -v repo_url="$PROXMOX_REPO_URL" '
         {
             print
+            if ($0 ~ /^source <\(curl -fsSL / && !curl_patched) {
+                print "curl() {"
+                print "  case \"${*: -1}\" in"
+                print "    */install/moltis-install.sh)"
+                print "      command curl \"$@\" | sed '\''s/read -r -p \"${TAB3}Would you like to install Docker for sandbox support? <y\\/N> \" prompt/if [[ -t 0 ]]; then read -r -p \"${TAB3}Would you like to install Docker for sandbox support? <y\\/N> \" prompt; else prompt=\"${MOLTIS_INSTALL_DOCKER:-no}\"; fi/'\''"
+                print "      ;;"
+                print "    *) command curl \"$@\" ;;"
+                print "  esac"
+                print "}"
+                curl_patched = 1
+            }
             if ($0 == "description" && !patched) {
                 q = sprintf("%c", 39)
                 print "pct exec \"$CTID\" -- bash -c \"cat > /usr/bin/update <<" q "MOLTIS_UPDATE_EOF" q

